frontend/src/layouts/JiraDashboard/JIRA/Jira.js

```diff
@@ -1,11 +1,13 @@
 import React from "react";
-import useAuthorize from "../JIRA_Authorize/useAuthorize";
```

Comment:
need to move all the components under components directory and not keep it in layouts directory